notebooks/plotly_roc_plot.py

```diff
@@ -294,6 +316,11 @@ def normalize(x, x_min, x_max):
                 y_data = np.array(this_data[task_idx][metric_name][alg_idx][y_key])
                 scores = np.array(this_data[task_idx][metric_name][alg_idx]["score"])
 
+                if alg_idx.lower() in ["hisp", "16h"]:
+                    scores = np.array(this_data[task_idx][metric_name][alg_idx]["n_nodes"])
+                    for i in range(1, len(this_data[task_idx][metric_name][alg_idx]["n_nodes"])-1): # first and last broken I think
```

Comment:
the score is not the number of nodes, it's the sum of pruning scores for HISP and the lambda-regularizer for 16h.